mandatory/2-function-creation.js

```diff
@@ -25,7 +37,13 @@ Write a function that:
 - removes any forward slashes (/) in each string
 - makes all strings all lowercase
 */
-function tidyUpStrings(arrayOfStrings) {}
+function tidyUpStrings(arrayOfStrings) {
+    let newArray = [];
+  for (let i = 0; i < arrayOfStrings.length; i++) {
+    newArray.push(arrayOfStrings[i].trim().replace("/", "").toLowerCase());
+  }
+  return newArray;
```

Comment:
Yes, good use of chaining functions here to make the code more concise.  It's personal preference as to how much chaining you do with multiple methods, but it's good that you know how to do it, so you have the choice.